horovod/common/__init__.py

```diff
@@ -37,11 +38,17 @@ def get_ext_suffix():
                              'mpi_lib' + get_ext_suffix()), mode=ctypes.RTLD_GLOBAL)
 
 
-def init():
+def init(mpi_init=True ,mpi_finalize=True):
     """A function that initializes Horovod.
     """
-    return MPI_COMMON_LIB_CTYPES.horovod_init()
+    return MPI_COMMON_LIB_CTYPES.horovod_init(ctypes.c_bool(mpi_init),ctypes.c_bool(mpi_finalize))
 
+def _close():  
+    """A function used by horovod to close Horovod during normal interpreter termination.
```

Comment:
close -> finalize.  Not sure what "normal interpreter termination" means - this will be called in case of an exception, too. Maybe just get rid of that part.